src/pages/media-kit/index.page.tsx

```diff
@@ -0,0 +1,233 @@
+import type { GetStaticProps, NextPage } from 'next'
+import Head from 'next/head'
+import { useTranslation } from 'next-i18next'
+import { serverSideTranslations } from 'next-i18next/serverSideTranslations'
+import { Page } from '../../components/Page'
+import { StyledLink } from '../../components/StyledLink'
+import EmbellishedLeft from './embellished/left.svg'
+import EmbellishedRight from './embellished/right.svg'
+import styles from './index.module.scss'
+
+const MediaKit: NextPage = () => {
+  const { t } = useTranslation(['media-kit', 'common'])
+
+  const sections: Array<{
+    title: string
+    description: string
+    download?: Record<'label' | 'link', string>
+    link?: Record<'label' | 'link', string>
+    assets: React.ReactElement
+  }> = [
+    {
+      title: t('logo.title'),
+      description: t('logo.description'),
+      download: {
+        label: t('logo.download'),
+        link: '/assets/media-kit/Nervos Logo.zip',
+      },
+      assets: (
+        <div>
+          {['light', 'dark'].map(type => (
+            <img
```

Comment:
Try to use the `next/Image` component as much as possible.

Reply:
Most `img` were replaced by `Image` in https://github.com/Magickbase/nervos-official-website/pull/359/commits/1e91da0fa14e27aa731890d4a173c9639e92d768 except the `typography` one because `Image` doesn't support auto-scale well.

Reply:
I tried it, providing `height: auto` in css is feasible, because its priority is higher than the value provided by attr.

Reply:
Setting `layout="responsive"` works

Commit: https://github.com/Magickbase/nervos-official-website/pull/359/commits/0581a774b3ca246a400149a614549650b51c3c81